### PR TITLE
Bugfix/pdf link

### PR DESCRIPTION
**Pull request by sairaghav**
Need to add a loader, but thought we will discuss on how to proceed with it together and create a loader component. Right now, only displaying message if pdf is being validated.
Also, facing CORS issue on most websites except arxiv.org. So let's just have that for now, maybe?

### CODE REVIEW

**Review by nMedic246 — COMMENTED** [review]
It seems to be working well.
A loader would be nice to have.

File: src/components/UploadModal/UploadModal.scss [hunk 1]
```diff
@@ -161,6 +161,14 @@
       }
     }
 
+    .modal-support-message {
+      @include typographyRegularXxs;
+      size: 10px;
```
Comment (vlahovivan) [inline src/components/UploadModal/UploadModal.scss]: what does this do?

Comment (sairaghav) [reply src/components/UploadModal/UploadModal.scss]: image
Just a message for notifying user that currently it's only arxiv.org

Comment (vlahovivan) [reply src/components/UploadModal/UploadModal.scss]: I know that, but I'm talking about the  `size: 10px` line, I don't think that's valid css

Comment (sairaghav) [reply src/components/UploadModal/UploadModal.scss]: will remove

**Review by vlahovivan — DISMISSED** [review]
It looks good, I think you can resolve conflicts and fix that minor thing I commented, and merge this to develop.

File: src/components/UploadModal/UploadModal.scss (hunk 1 above)
Comment (sairaghav) [reply src/components/UploadModal/UploadModal.scss]: is this ok to merge now?

Comment (vlahovivan) [reply src/components/UploadModal/UploadModal.scss]: yes, but we have new conflicts now :sweat_smile:

**Review by vlahovivan — APPROVED** [review]

**Pull request merged**